specification/cluster/put_component_template/ClusterPutComponentTemplateRequest.ts

```diff
@@ -87,7 +87,7 @@ export interface Request extends RequestBase {
     /**
      * The template to be applied which includes mappings, settings, or aliases configuration.
      */
-    template: IndexState
+    template: IndexTemplateMapping
```

Comment:
Both component templates and composable/index templates define the `template` field. In the ES code, they both point to the same class and are thus parsed in the same way, so we can/should use the same class in the spec too. (just explaining why I made this change)